server/src/test/java/io/crate/integrationtests/UnionIntegrationTest.java

```diff
@@ -327,4 +327,18 @@ public void testSimpleUnionDistinct() {
         execute("select a from x union distinct select a from y order by a");
         assertThat(printedTable(response.rows()), is("1\n2\n3\n5\n"));
     }
+
+    @Test
+    public void test_union_of_two_null_columns_is_null() {
```

Comment:
had it for debugging full flow - I guess we are good with the planner test and this can be removed?